ranch/ranch.go

```diff
@@ -114,7 +114,7 @@ type acquireRequestPriorityKey struct {
 //     requestID - request ID to get a priority in the queue
 // Out: A valid Resource object on success, or
 //      ResourceNotFound error if target type resource does not exist in target state.
-func (r *Ranch) Acquire(rType, state, dest, owner, requestID string) (*crds.ResourceObject, error) {
+func (r *Ranch) Acquire(rType, state, dest, owner, requestID string) (*crds.ResourceObject, time.Time, error) {
```

Comment:
can you update the docstring to include this new return value?

I also still have a slight preference for this function calculating the duration and returning it itself, but I won't hold my approval over this.

Reply:
docstring is updated.

Also tried to calculate the duration here and did not make it work (need to mock another `now`).